apollo-adminservice/src/main/resources/application-nacosDiscovery.properties

```diff
@@ -0,0 +1,6 @@
+eureka.client.enabled=false
```

Comment:
I tested the profile named as `nacos-discovery` which works, so I think we may just use `nacos-discovery` as the profile name, whichi is the same as maven profile.